.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: "Test"
+
+on:
+  push:
```

Comment:
Please specify a branch (or branches) here. Using `push` without a branch and `pull_request` is usually redundant. Typically only the trunk branch is needed here, but others can be added if there are other long-lived branches in the repo that require that this be run when pushed to.

By adding a branch specifier, this cuts down on GHA minutes usage from redundant builds.